pypeit/calibrations.py

```diff
@@ -381,13 +384,17 @@ def get_arc(self):
         # Perform a check on the files
         self.check_calibrations(raw_files)
 
+        # If a manual spatial flexure is requested for any of the frames, collect it now
+        manual_spat_flexure = self.fitstbl.get_shifts(raw_index)
+
         # Otherwise, create the processed file.
         msgs.info(f'Preparing a {frame["class"].calib_type} calibration frame.')
         self.msarc = buildimage.buildimage_fromlist(self.spectrograph, self.det,
                                                     self.par['arcframe'], raw_files,
                                                     bias=self.msbias, bpm=self.msbpm,
                                                     dark=self.msdark, calib_dir=self.calib_dir,
```

Comment:
Found a bug. Obviously not introduced here.

```suggestion
                                                    dark=self.msdark, slits=self.slits, calib_dir=self.calib_dir,
```